rust/client/src/lib.rs

```diff
@@ -215,12 +215,12 @@ impl BpxClient {
     }
 
     /// Returns a reference to the `VerifyingKey` used for request verification.
-    pub fn verifier(&self) -> &VerifyingKey {
+    pub const fn verifier(&self) -> &VerifyingKey {
         &self.verifier
     }
 
     /// Returns a reference to the underlying HTTP client.
-    pub fn client(&self) -> &reqwest::Client {
+    pub const fn client(&self) -> &reqwest::Client {
```

Comment:
"Constify" some methods where possible. It should be possible to "constify" more in the future